src/Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.2022/Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.2022.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <RootNamespace>Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas._2022</RootNamespace>
```

Comment:
"_2022" seems like a bit odd of a root namespace - is this standard for this codebase?